ep/build-velox/src/get_velox.sh

```diff
@@ -16,8 +16,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=update
+VELOX_REPO=https://github.com/liujiayi771/velox.git
+VELOX_BRANCH=update-decimal-sum
```

Comment:
@liujiayi771 Could you help open a PR to https://github.com/oap-project/velox with this branch?